desktop-widgets/mainwindow.cpp

```diff
@@ -581,7 +581,8 @@ void MainWindow::enableShortcuts()
 void MainWindow::showProfile()
 {
 	enableShortcuts();
-	profile->plotCurrentDive();
+//	profile->plotCurrentDive();
```

Comment:
Any reason to keep this as a comment? I think this is wrong, as it will plot the currently shown dive. However, that is the planned dive when exiting the planner and thus is, in a sense, not valid anymore.